### PR TITLE
Ride dialog

### PR DESCRIPTION
**Pull request by jasiek-net**

### CODE REVIEW

File: src/controllers/ride.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+/**
+ * Ride Booking
+ */
+
+@Route('ride')
+export class RideController {
+
+  /**
+   * Create new ride booking
```
Comment (michalmikolajczyk) [inline src/controllers/ride.ts]: I don't really understand what is that `ride` entity. Is that a booking?

File: src/controllers/ride.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+   * @param {RideBody} object with booking info
```
Comment (michalmikolajczyk) [inline src/controllers/ride.ts]: if it is booking, let's name it `booking` throughout the apps, and not `ride`

File: src/mailer/index.ts [hunk 3]
```diff
@@ -3,6 +3,7 @@ import {
   signinMessage,
   resetMessage,
 } from './templates';
+const ride = process.env.VG_RIDE || 'viking.garage.app@gmail.com';
```
Comment (michalmikolajczyk) [inline src/mailer/index.ts]: @jonzee that env variable name is enigmatic, let's change it to something clear like `VG_BOOKING_EMAIL` or sth

File: src/services/ride/ride.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    endDate,
+    equipment,
+    price,
+    total,
+    currency,
+  };
+
+  db['ride'].create(newRide)
+    .then(user => rideEmail(newRide)
+      .then(() => res.status(200).json({ msg: 'Booking ride successfully' }))
```
Comment (michalmikolajczyk) [inline src/services/ride/ride.ts]: check grammar please

**Review by michalmikolajczyk — CHANGES_REQUESTED** [review]
I think that `ride` and `booking` are kind of mixed here. I would prefer to use the work `booking` for the bookings (Ride reservations). Can we change that please?

**Review by michalmikolajczyk — APPROVED** [review]

**Comment by michalmikolajczyk** [comment]
@jonzee prośba o odniesienie się do komentarzy

File: src/controllers/ride.ts (hunk 1 above)
Comment (jasiek-net) [reply src/controllers/ride.ts]: It's more like contact form, I'll update this before merge

**Pull request merged**